src/field.cpp

```diff
@@ -2071,6 +2093,23 @@ void map::player_in_field( player &u )
             }
             break;
 
+        case fd_fungicidal_gas:
+            // Fungicidal gas makes you cough.
+            // Thick fungicidal gas has a chance to poison you.
```

Comment:
What if you've crossed a fungal threshold like mycus or marloss? Shouldn't it be doing extra damage/poison to you?